Added criteria for backend takehome as well (#16)
Codecademy/engineering-competencies: interviews/README.md

# Takehome Grading

Hi!

> This is our grading rubric for takehome projects.

Reviewer: ```suggestion
These are our grading guidelines for takehome projects.
```

Thanks for taking a look!

> These are the scores by which our engineering team evaluate backend takehome assignments.

Reviewer: ```suggestion
These are the criteria our engineering team uses to evaluate takehome assignments.
```

> We _don't_ establish a hard minimum of score for any opening -- these scores are a general rubric, not a hard rule.

Reviewer: ```suggestion
We don't set a minimum score for any opening, and although we take these criteria very seriously, we also reserve the right to use considerations beyond these rubrics in our decision making.
```

## Rubrics

> 

Reviewer: Insert:

> Here are the specific grading rubrics we use to evaluate each of our takehome assignments:

- [Intern Backend Takehome](./frontend-takehome.md)
- [Intern Frontend Takehome](./backend-takehome.md)

## Why?

We believe in openness and transparency in our engineering practices.

> There must be _some_ criteria by which we evaluate takehome submissions, and if candidates are meant to be taking them, we'd like them to be clear to everyone.

Reviewer: I don't understand what "if candidates are meant to be taking them" means.

We also welcome any feedback you have on these!

> ## Takehome Assignment Specifications

Reviewer: I'm not sure whether this section belongs here. This section seems to be about what makes a good takehome assignment, whereas the rest of this document describes how we evaluate takehome submissions.

Author: 🤔 interesting. Filed a followup issue to discuss: #18
